### PR TITLE
Changed FieldsOfStudySelectField to use fullWidth: true

### LINKED ISSUE #580: Field of study needs more space since some fields show a lot of text

### PR DESCRIPTION
**Pull request by aliceriot**
#### What are the relevant tickets?

closes #580 

#### What's this PR do?

`fullWidth: false` -> `fullWidth: true`

#### How should this be manually tested?

Visit `/users/` or `/profile/education` and open an education form with the fields of study select field (i.e. not high school). Confirm the field is a bit wider.

### CODE REVIEW

**Comment by noisecapella** [comment]
This is a good change but I'm wondering if we just want to give field of study its own entire line. There are certain lines that don't have nearly enough space: 

screenshot from 2016-06-22 14-40-14

The text shown in this line here is only half as much as is in the field.

Thoughts @pdpinch @aliceriot @roberthouse54?

**Comment by roberthouse54** [comment]
I agree. Move it to its own line.

**Comment by aliceriot** [comment]
cool, should we move graduation date down to the next line next to 'school name'?

**Comment by roberthouse54** [comment]
Put it next to the school name... or if the school name field is also long, let's move it under school name.

**Comment by aliceriot** [comment]
here's what it looks like with field of study full-width:

full-width

I think the graduation date field looks a lil awkward over there - we could make the school name field be 2/3 of a line and the graduation date 1/3 instead of what is here (50/50).

**Comment by aliceriot** [comment]
here's what it looks like with the school name 3/4 of a line and graduation date 1/4:

full-width-entered

File: static/js/components/inputs/FieldsOfStudySelectField.js [hunk 1]
```diff
@@ -21,7 +21,7 @@ export default class FieldsOfStudySelectField extends React.Component {
   static autocompleteStyleProps = {
     menuStyle: {maxHeight: 300, width: 600},
```
Comment (noisecapella) [inline static/js/components/inputs/FieldsOfStudySelectField.js]: Can you remove `width: 600` here?

File: static/js/components/inputs/FieldsOfStudySelectField.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ export default class FieldsOfStudySelectField extends React.Component {
...
     listStyle: {width: '100%'},
```
Comment (noisecapella) [inline static/js/components/inputs/FieldsOfStudySelectField.js]: Can you remove this line too, and add `menuHeight: 300`? Basically I'm looking to make it the same as `SelectField.autocompleteStyleProps`, and to fix the field I forgot about before

**Comment by aliceriot** [comment]
@noisecapella I just updated the `autocompleteStyleProps`, does it look ok now? I think it fixed a console warning that was introduced recently (currently present on master).

**Comment by noisecapella** [comment]
Yes, actually fields of study wasn't working for me at all due to the missing `menuHeight`. Looks great now!

**Comment by noisecapella** [comment]
Is there some way to get the graduation date to stick to the right side of its `Cell`? Just a nitpick, looks great :+1:

**Comment by aliceriot** [comment]
If I put `text-align: right` on that div we get this:

right

@roberthouse54 what do you think?

**Comment by roberthouse54** [comment]
No, we do not want that aligned to the right. It should look basically like this:
d8e4e29e-392c-11e6-8633-c4bb271acfe6

**Comment by noisecapella** [comment]
Works for me

**Pull request merged**